openmc/statepoint.py

```diff
@@ -458,15 +458,22 @@ def version(self):
                 self._f['version_release'].value)
 
     @property
+    def summary(self):
+        return self._summary
+
+    @property
     def with_summary(self):
-        return self._with_summary
+        if self.summary is None:
+            return False
+        else:
+            return True
```

Comment:
Oooh, I like that a lot! I didn't know that was possible. I'm now using this for several of the properties for the ``StatePoint`` class.